keyboards/jlw/brian_the_keyboard/keymaps/default/keymap.c

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 sporkus
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#define HM_A	LSFT_T(KC_A)
+#define HM_S	LALT_T(KC_S)
+#define HM_D	LGUI_T(KC_D)
+#define HM_F	LCTL_T(KC_F)
+#define HM_J	RCTL_T(KC_J)
+#define HM_K	RGUI_T(KC_K)
+#define HM_L	RALT_T(KC_L)
+#define HM_Z	LSFT_T(KC_Z)
+#define HM_SLSH	LSFT_T(KC_SLSH)
+#define HM_QUOT	RSFT_T(KC_QUOT)
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+         KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,   KC_P,
+             KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,
+                 KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,
+                 KC_LCTRL        MO(1),  KC_ENT, KC_SPC, MO(2),          KC_RALT
+    ),
+
+    [1] = LAYOUT(
+         _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+              _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,            _______,  _______,  _______,  _______,            _______
+    ),
+
+    [2] = LAYOUT(
+         _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+              _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,            _______,  _______,  _______,  _______,            _______
```

Comment:
No need to hardcode filler `KC_TRNS` layers. For non-dynamic keymaps it's pointless, and dynamic keymaps will automatically fill in "missing" layers at compile-time up to the maximum set in `dynamic_keymap.layer_count`.

...also don't forget to compile your keymaps before opening a pull request.
```suggestion
                 KC_LCTRL,       MO(1),  KC_ENT, KC_SPC, MO(2),          KC_RALT
```

Reply:
Absolutely correct, thank you for bringing that to my attention. This has been resolved in the most recent commit